src/core/client/stream/tabs/Comments/Comment/UserBanPopover/UserBanPopoverContainer.tsx

```diff
@@ -38,14 +34,6 @@ const UserBanPopoverContainer: FunctionComponent<Props> = ({
   const banUser = useMutation(BanUserMutation);
   const { localeBundles } = useCoralContext();
 
-  // Not checking for multisite here due to permission issues.
-  // We shouldn't have this enabled unless we're already multisite
-  // anyways. Also, if we send site ID's and multisite is off, the
-  // backend will handle this.
-  const moderationScopesEnabled = settings.featureFlags.includes(
```

Comment:
We don't need any checks here because site moderators do not see this `UserBanPopoverContainer` in the stream; only admins or organization moderators do. This is also why the alternative description below has been removed.